src/main/java/pl/touk/sputnik/engine/score/ScoreStrategyFactory.java

```diff
@@ -0,0 +1,48 @@
+package pl.touk.sputnik.engine.score;
+
+import lombok.extern.slf4j.Slf4j;
+import org.jetbrains.annotations.NotNull;
+import pl.touk.sputnik.configuration.Configuration;
+import pl.touk.sputnik.configuration.GeneralOption;
+
+import static org.apache.commons.lang3.Validate.notBlank;
+
+@Slf4j
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.